expressEngine/lib/validationHandler/validateRequired.js

```diff
@@ -0,0 +1,13 @@
+class validateRequired {
```

Comment:
Why we are creating class for it?
I think creating a simple method and exporting it will do the job.
Let me know if you have any different view.

Reply:
Now I have updated it.